### DISCUSSION TITLE
What is the purpose of the dummy app?

### DISCUSSION THREAD

**Discussion by bdjohnson529** [excid3/noticed#152] [General]
Hi guys,

Quick question here. Is the rails app contained in `/tests/dummy` intended to work? I thought maybe it was intended to demonstrate the functionality of the library, but I have been unable to get it work.

Maybe it's used for a different purpose?

**Comment by excid3** [comment]
It's used for the test suite. 👍

**Comment by bdjohnson529** [comment]
Thanks Chris! This gem is awesome.
